src/python_bindings/py_algorithm.h

```diff
@@ -47,6 +87,77 @@ class PyAlgorithmBase {
     void LoadData();
 
     pybind11::int_ Execute(pybind11::kwargs const& kwargs);
+    std::unordered_map<std::string, std::string> GetOpts() {
+        std::unordered_map<std::string_view, boost::any> opt_ = algorithm_->GetOptValues();
+        std::unordered_map<std::string, std::string> res;
+        // for (auto& i:converters){
+        //     std::cout<<i.first<<" "<<'\n';
+        // }
+        for (auto& i : opt_) {
+            if (i.first == "table" || i.first == "input_format") {
```

Comment:
Skipping the table option is fine, but please use the constant from config::names

Reply:
If the conversion is implemented based on type index, converting `input_format` will be just another pair in a map, thus not requiring much effort to handle it too.